lib/OpenQA/WebAPI/Controller/API/V1/JobTemplate.pm

```diff
@@ -115,16 +115,23 @@ Returns a YAML template representing the job group(s).
 sub schedules {
     my $self = shift;
 
-    my $yaml = $self->get_job_groups($self->param('id'));
+    my $yaml = $self->get_job_groups($self->param('id'), $self->param('name'));
+
+    # Re-indent with group names at the toplevel in case of multiple groups
+    if (scalar $yaml > 1) {
+        foreach my $group (keys %$yaml) {
+            $yaml->{$group} = "$group:\n" . $yaml->{$group} =~ s/(.+)\n/  $1\n/gr;
```

Comment:
That is really creative, but here i would actually use parentheses to make the operator precedence more obvious.